### PR TITLE
chore_: initial codeowners

### PR DESCRIPTION
**Pull request by igor-sirotin**
No default code owners for now.
Defined ownership for CI-related code.
Defined my ownership for some folders.

Expecting this file to grow further later.

### CODE REVIEW

**Review by dlipicar — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]

**Comment by igor-sirotin** [comment]
@status-im/devops can you please take a look if this works for you?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# No default owner
+
+# CI-related changes
+Makefile       @status-im/devops @status-im/status-go-guild
+/nix/          @status-im/devops
+/_assets/ci/   @status-im/devops
```
Comment (jakubgs) [inline .github/CODEOWNERS]: You're forgetting:

- `shell.nix`
- `default.nix`
- `/_assets/scripts/`
- `/_assets/hooks/`
- `/_assets/build/`

**Review by jakubgs — APPROVED** [review]
Reasonable.

File: .github/CODEOWNERS (hunk 1 above)
Comment (igor-sirotin) [reply .github/CODEOWNERS]: Will add 👍

**Pull request merged**